source/slang/syntax-base-defs.h

```diff
@@ -195,6 +195,40 @@ SYNTAX_CLASS(ThisTypeSubstitution, Substitutions)
         return sourceType->GetHashCode();
     }
     )
+    END_SYNTAX_CLASS()
```

Comment:
Visual Studio likes to mess up the indenting in these files, unfortunately, so you have to watch that you don't introduce spurious diffs on the existing declarations.